common/journal_entries/00_sick_man.txt

```diff
@@ -100,8 +100,10 @@ je_sick_man_economy = {
 			filter = {
 				is_incorporated = yes
 			}
-			has_building = building_urban_center
-			percent >= 0.75
+			state_has_building_type_levels = {  
+				target = bt:building_urban_center
+				value >= 3
+			}
```

Comment:
This reads like you just need one state with urban center at or above level 3 to get this. Is that right?

Reply:
Yes, it was a mistake when working with danarca to enact it.